features/user-store/org.wso2.carbon.identity.user.store.configuration.server.feature/resources/build.properties

```diff
@@ -0,0 +1,2 @@
+custom = true
+root.user-store=conf
```

Comment:
formatting issue. add space before and after "="

Reply:
fixed